src/citrine/informatics/design_spaces/product_design_space.py

```diff
@@ -11,56 +12,75 @@
 
 
 class ProductDesignSpace(Resource['ProductDesignSpace'], DesignSpace):
-    """[ALPHA] An outer product of univariate dimensions, either continuous or enumerated.
+    """[ALPHA] An outer product of design spaces.
+
+    Factors can be other design spaces and/or univariate dimensions.
 
     Parameters
     ----------
     name:str
         the name of the design space
     description:str
         the description of the design space
+    subspaces: List[Union[UUID, DesignSpace]]
+        the list of subspaces to combine, either design spaces defined in-line
+        or UUIDs that reference design spaces on the platform
     dimensions: list[Dimension]
         univariate dimensions that are factors of the design space; can be enumerated or continuous
 
     """
 
     _response_key = None
 
-    uid = properties.Optional(properties.UUID, 'id', serializable=False)
-    name = properties.String('config.name')
-    description = properties.Optional(properties.String(), 'config.description')
-    dimensions = properties.List(properties.Object(Dimension), 'config.dimensions')
-    typ = properties.String('config.type', default='Univariate', deserializable=False)
-    status = properties.String('status', serializable=False)
-    status_info = properties.Optional(
-        properties.List(properties.String()),
-        'status_info',
-        serializable=False
-    )
-    archived = properties.Boolean('archived', default=False)
-    experimental = properties.Boolean("experimental", serializable=False, default=True)
-    experimental_reasons = properties.Optional(
-        properties.List(properties.String()),
-        'experimental_reasons',
-        serializable=False
+    subspaces = properties.List(properties.Union(
+        [properties.UUID, properties.Object(DesignSpace)]
+    ), 'config.subspaces', default=[])
+    dimensions = properties.Optional(
+        properties.List(properties.Object(Dimension)), 'config.dimensions'
     )
+    typ = properties.String('config.type', default='ProductDesignSpace', deserializable=False)
+    # Product design spaces may not be embedded in other subspaces, hence status is required
```

Comment:
Is this enforced by the backend?

Reply:
No I don't think so. I also don't think we strictly enforce "no graph predictors inside graph predictors." But in both cases we don't support it, it might not work, and it provides no additional functionality. So I changed the comment to "should not."